### PR TITLE
Added send-report documentation.

### PR DESCRIPTION
**Pull request by elongl**

### CODE REVIEW

File: docs/guides/data-and-dbt-tests-report-ui.mdx [hunk 1]
```diff
@@ -63,11 +66,13 @@ The rows are expandable, and on expansion show information about the last runs o
...
 - Time
 
-<img src="https://res.cloudinary.com/mintlify/image/upload/v1659304885/elementary/Test-Runs-video-demo_ubxssd.gif" alt="Test runs demo" />
+<img
+  src="https://res.cloudinary.com/mintlify/image/upload/v1659304885/elementary/Test-Runs-video-demo_ubxssd.gif"
+  alt="Test runs demo"
+/>
 
-## Configure Slack report
+## Exporting the report
```
Comment (Maayan-s) [inline docs/guides/data-and-dbt-tests-report-ui.mdx:74]: I would name it "Sharing the report", as this is the user's perspective. 
Also, this is what we want to incentivize them to do.

File: docs/guides/data-and-dbt-tests-report-ui.mdx [hunk 2]
```diff
@@ -63,11 +66,13 @@ The rows are expandable, and on expansion show information about the last runs o
...
 
-The data tests report UI can be sent via Slack when you run `edr monitor send-report`.
-Configure Slack integration by creating an app for Elementary and adding the `Token/Webhook` to the `config.yml`.
+The data tests report UI can be sent via Slack, Amazon S3, or Google Cloud Storage when you run `edr monitor send-report`.
```
Comment (Maayan-s) [inline docs/guides/data-and-dbt-tests-report-ui.mdx:76]: Elementary offers 3 methods that makes it easy to share the report with others:
1. Host on ...
2. Host on ... 
3. Send...

Refer to..."

I think that it's a bit useless to provide the command without instructions. 
Right? Better to refer or add the full explanation.

File: docs/quickstart/generate-report-ui.mdx [hunk 3]
```diff
@@ -120,12 +120,12 @@ The command will use the provided connection profile to access the data warehous
 
 ---
 
-## Send Report to Slack
+## Send Report to External Platform
```
Comment (Maayan-s) [inline docs/quickstart/generate-report-ui.mdx:123]: You can make a 'Sharing the report' snippet. 
(I explained on the other docs branch you created)

File: docs/quickstart/send-report-ui.mdx [hunk 4]
```diff
@@ -0,0 +1,104 @@
+---
+title: "Send observability report"
+---
+
+import InstallCLI from "../snippets/install-cli.mdx";
+import InstallDbtPackage from "../snippets/install-dbt-package.mdx";
+
```
Comment (Maayan-s) [inline docs/quickstart/send-report-ui.mdx:7]: You imported the components but didn't include them.
Refer to the 'generate-report-ui' page to see how to do it.

File: docs/quickstart/send-report-ui.mdx [hunk 5]
```diff
@@ -0,0 +1,104 @@
+---
+title: "Send observability report"
+---
```
Comment (Maayan-s) [inline docs/quickstart/send-report-ui.mdx:3]: Share data observability report? 
Sharing the report?

File: docs/quickstart/send-report-ui.mdx [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+import InstallCLI from "../snippets/install-cli.mdx";
+import InstallDbtPackage from "../snippets/install-dbt-package.mdx";
+
+Distribute the report by sending it to external platforms using the command `edr monitor send-report`.  
+Each specified platform in the command will be sent a report.
```
Comment (Maayan-s) [inline docs/quickstart/send-report-ui.mdx:9]: "You can specify more than one platform in the command, and the report will be sent to all."

File: docs/quickstart/send-report-ui.mdx [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+## Slack
+
+After you [setup a Slack app and token](../integrations/slack#slack-integration-setup) you can run the command:
+
+```shell
+edr monitor send-report --slack-token <SLACK_TOKEN> --slack-channel-name <CHANNEL_NAME>
+```
+
+A report will be sent to your Slack channel.
+
```
Comment (Maayan-s) [inline docs/quickstart/send-report-ui.mdx:21]: Maybe refer here to the multiple channels and subscribers config we have? 
We want to promote it.

File: docs/quickstart/send-report-ui.mdx [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+### Configuration
+
+If you prefer to use `config.yml` file, you can add the following configuration instead of the CLI arguments:
+
+```yml
+google:
+  service_account_path: [ Google service account path ]
+  gcs_bucket_name: [ GCS bucket name ]
+update_bucket_website: [ true | false ]
+```
```
Comment (Maayan-s) [inline docs/quickstart/send-report-ui.mdx:104]: I think this part about the config of the new integrations is great!
I would add a section in the bottom saying something like:
"Want to share the report with a different method or consult on deployment? Reach out to us on Slack.."

This is the deploy to prod part, we need to make it as easy as possible.

File: docs/quickstart/send-report-ui.mdx (hunk 5 above)
Comment (Maayan-s) [inline docs/quickstart/send-report-ui.mdx:3]: This is a new page, so you need to add it to `mint.config.json`

File: docs/guides/data-and-dbt-tests-report-ui.mdx (hunk 1 above)
Comment (elongl) [reply docs/guides/data-and-dbt-tests-report-ui.mdx:74]: Totally agree!

File: docs/guides/data-and-dbt-tests-report-ui.mdx (hunk 2 above)
Comment (elongl) [reply docs/guides/data-and-dbt-tests-report-ui.mdx:76]: What do you mean? I do that in the line below.
image

File: docs/quickstart/send-report-ui.mdx (hunk 4 above)
Comment (elongl) [reply docs/quickstart/send-report-ui.mdx:7]: I deleted them since they're not needed.

File: docs/quickstart/send-report-ui.mdx (hunk 7 above)
Comment (elongl) [reply docs/quickstart/send-report-ui.mdx:21]: Wouldn't it be confusing to the user to see other configuration values given that it's unrelated to the report?

File: docs/quickstart/share-report-ui.mdx [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+If you prefer to use `config.yml` file, you can add the following configuration instead of the CLI arguments:
+
+```yml
+google:
+  service_account_path: [ Google service account path ]
+  gcs_bucket_name: [ GCS bucket name ]
+update_bucket_website: [ true | false ]
+```
+
+## Missing platform?
```
Comment (Maayan-s) [inline docs/quickstart/share-report-ui.mdx:103]: Let's change to "Missing something?", more friendly IMO

File: docs/understand-elementary/elementary-report-ui.mdx [hunk 10]
```diff
@@ -72,7 +87,8 @@ The rows are expandable, and on expansion show information about the last runs o
 
 ## Configure Slack report
```
Comment (Maayan-s) [inline docs/understand-elementary/elementary-report-ui.mdx:88]: Change headline

File: docs/quickstart/send-report-ui.mdx (hunk 4 above)
Comment (Maayan-s) [reply docs/quickstart/send-report-ui.mdx]: The only dilemma is if we need to look at the page as "part of a flow" or "all you need to setup this e2e".
This is what Or mentioned this morning.
Anyway, we can decide on it later. I'm fine with this for now.

**Comment by Maayan-s** [comment]
2 small changes and you can mark this as ready for the release.

**Comment by Maayan-s** [comment]
This is ready for merge

**Pull request merged**